### DISCUSSION TITLE
NFTs for Early DAN Supporters: A Token of Appreciation and Potential Benefits

### DISCUSSION THREAD

**Discussion by maxwilliamdev** [fiatrete/SDCN-Stable-Diffusion-Computing-Network#74] [Ideas]
I'm planning to have some fun with NFTs related to DAN,  I often recommend to my friends. We've had many discussions about AI and what DAN could accomplish in the future. I want to create an NFT as a token of appreciation for those early supporters and contributors to DAN. I haven't figured out the exact benefits that this NFT will offer, but for now, I'm personally giving these NFTs as gifts. I believe that such NFTs will ultimately benefit the DAN project.

For now, I'll handle all of this on my own.

If anyone has any suggestions, feel free to share them with me directly.

**Comment by fiatrete** [comment]
Have fun with it. When your NFTs come out, don't forget to save me one: )

**Reply by voltff** [reply]
i need one

**Reply by maxwilliamdev** [reply]
coming soon 😄

**Comment by kaol2046** [comment]
brainstorm LOL

![image](https://user-images.githubusercontent.com/2410035/236088831-60212205-3767-42f9-851f-d6acbf3bf143.png)
![tsanchez_facepfpcyberpunk_cool_boy_show_mouth_5e2b13ec-108e-47ec-9310-249e67d51ca6](https://user-images.githubusercontent.com/2410035/236085293-3d997f6c-387e-4fb1-a09c-868851f219d0.png)
![head](https://user-images.githubusercontent.com/2410035/236085383-ab9c89f3-36d2-4232-a188-bd16d12624cd.gif)

**Comment by maxwilliamdev** [comment]
It looks a bit like Apple VR!

**Comment by maxwilliamdev** [comment]
NFT has been uploaded, hoping that more friends can acquire these meaningful NFTs.

[OpenDAN Star NFT](https://opensea.io/collection/opendanstar)
